tests/vocabularies/vocabularies_service_get_items_tests.py

```diff
@@ -0,0 +1,269 @@
+                                "name": {
+                                    "nl": "Antwerpen",
+                                    "fr": "Anvers"
+                                }
+                            }
+                        },
+                    ],
+                },
+                {
+                    "_id": "sad-keywords",
+                    "items": [
+                        {
+                            "name": "WALLOON BRABANT",
+                            "qcode": "WALLOON BRABANT",
+                            "is_active": True,
+                            "translations": {
+                                "name": {
+                                    "nl": "Waals-Brabant",
+                                    "fr": "Brabant wallon"
+                                }
+                            }
+                        },
+                        {
+                            "name": "BIOGRAPHY",
+                            "qcode": "BIOGRAPHY",
+                            "is_active": True,
+                            "translations": {
+                                "name": {
+                                    "nl": "Biografie",
+                                    "fr": "Biographie"
+                                }
+                            }
+                        },
+                        {
+                            "name": "WEATHER",
+                            "qcode": "WEATHER",
+                            "is_active": True,
+                            "translations": {
+                                "name": {
+                                    "nl": "Weer",
+                                    "fr": "Météo"
+                                }
+                            }
+                        },
+                    ],
+                },
+            ],
+        )
+
+    def test_search_by_id(self):
+        items = superdesk.get_resource_service('vocabularies').get_items(
+            _id='funny-keywords'
+        )
+        self.assertEqual(len(items), 5)
+
+        items = superdesk.get_resource_service('vocabularies').get_items(
+            _id='sad-keywords'
+        )
+        self.assertEqual(len(items), 3)
+
+        items = superdesk.get_resource_service('vocabularies').get_items(
+            _id='not-found-id'
+        )
+        self.assertIsNone(items)
+
+    def test_search_by_qcode(self):
+        items = superdesk.get_resource_service('vocabularies').get_items(
+            _id='funny-keywords',
+            qcode='PREVIEW'
+        )
+        self.assertEqual(
+            items,
+            [{
+                "name": "PREVIEW",
+                "qcode": "PREVIEW",
+                "is_active": True,
+                "translations": {
+                    "name": {
+                        "nl": "Voorbericht",
+                        "fr": "Avant-Papier"
+                    }
+                }
+            }]
```

Comment:
imo would be good if the items are ready for use, so it would be without `is_active` and with `scheme` already set so you could then just do `item['subject'].extend(items)`